prospector/config.yaml

```diff
@@ -0,0 +1,50 @@
+# The vulnerability identifier to look for
+cve_id: ""
+
+# The related repository URL
+repository: ""
```

Comment:
the stuff that one would change for each invocation, should not be in the configuration file